operations/transaction-service/src/config/client-address-mapping.example.json

```
@@ -0,0 +1,12 @@
+{
+  "clientIdA": {
+      "walletAddress": "<address-for-this-client>",
```

```suggestion
      "publicWalletAddress": "<address-for-this-client>",
```
Let's use the term`publicWalletAddress`, which will be more meaningful.